### PR TITLE
Device Type: Add Schneider Electric SpaceLogic™ AS-P-NL specification…

### PR DESCRIPTION
**Pull request by netravnen**
… and front image

### CODE REVIEW

**Comment by netravnen** [comment]
Made the power power type "other" to resolve the power requirement

**Review by DanSheps — APPROVED** [review]

File: elevation-images/Schneider Electric/SpaceLogic-AS-P-NL.front.png
Comment (danner26) [inline elevation-images/Schneider Electric/SpaceLogic-AS-P-NL.front.png:1]: We need these to be cleaned up. The text should be removed.

File: device-types/Schneider Electric/SpaceLogic-AS-P-NL.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+manufacturer: Schneider Electric
+model: SpaceLogic AS-P-NL
+slug: schneider-electric-spacelogic-as-p-nl
+part_number: SXWASPXXX10002
+is_full_depth: false
+u_height: 0
```
Comment (danner26) [inline device-types/Schneider Electric/SpaceLogic-AS-P-NL.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Schneider Electric/SpaceLogic-AS-P-NL.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    type: usb-mini-b
+  - name: serial2
+    label: Host
+    type: usb-micro-a
+power-ports:
+  - name: power
+    label: Power
+    type: other
+    maximum-draw: 10
+comments: '[SpaceLogic™ AS-P-NL Hardware Installation](https://ecostruxure-building-help.se.com/bms/Topics/Show.castle?id=13809)'
```
Comment (danner26) [inline device-types/Schneider Electric/SpaceLogic-AS-P-NL.yaml:31]: Please move this comment up towards before interfaces

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Schneider Electric/SpaceLogic-AS-P-NL.yaml (hunk 2 above)
Comment (netravnen) [reply device-types/Schneider Electric/SpaceLogic-AS-P-NL.yaml:31]: @danner26 I would like you to look into incorporating this check as part of the "pre-commit run" book the repository currently includes.

I rely on the `pre-commit run` to be able comply with expectations for PRs before submitting them here on GH. ;-)

File: device-types/Schneider Electric/SpaceLogic-AS-P-NL.yaml (hunk 1 above)
Comment (netravnen) [reply device-types/Schneider Electric/SpaceLogic-AS-P-NL.yaml:7]: > Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.
> 
> 1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
> 2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
> 
> If you have any questions, please let us know :)

Done ✅

File: elevation-images/Schneider Electric/SpaceLogic-AS-P-NL.front.png
Comment (netravnen) [reply elevation-images/Schneider Electric/SpaceLogic-AS-P-NL.front.png:1]: > The text should be removed

Should I remove the image?

Comment (netravnen) [reply elevation-images/Schneider Electric/SpaceLogic-AS-P-NL.front.png:1]: ... removed the image

File: device-types/Schneider Electric/SpaceLogic-AS-P-NL.yaml (hunk 2 above)
Comment (danner26) [reply device-types/Schneider Electric/SpaceLogic-AS-P-NL.yaml]: I agree, not entirely sure how I will work that out but ill see what I can do!

**Review by danner26 — APPROVED** [review]

**Pull request merged**